File: lab 9/rest-service/src/main/java/com/example/restservice/MyData.java
package com.example.restservice;

import java.net.Inet4Address;
import java.net.UnknownHostException;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

public class MyData {
    public static void info() throws UnknownHostException {
        System.out.println("Filip Wiśniewski, 260406");
        System.out.println("Szymon Bąk, 260431");
        System.out.println(LocalDateTime.now().format(DateTimeFormatter.ofPattern("dd MMMM HH:mm:ss")));
        System.out.println(System.getProperty("java.version"));
        System.out.println(System.getProperty("user.name"));
        System.out.println(Inet4Address.getLocalHost().getHostAddress());
        System.out.println("Witaj w świecie Javy \uD83D\uDE00\n");
    }
}

File: lab 13/sender/sender/MyData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Net;
using System.Net.Sockets;

namespace WcfClient
{

    public class MyData
    {
        public static void Info()
        {
            Console.WriteLine("Filip Wiśniewski, 260406");
            Console.WriteLine("Szymon Bąk, 260431");
            Console.WriteLine(DateTime.Now.ToString("dd MMMM HH:mm:ss"));
            Console.WriteLine(Environment.OSVersion.VersionString);
            Console.WriteLine(Environment.UserName);
            Console.WriteLine(Environment.Version.ToString());
            var host = Dns.GetHostEntry(Dns.GetHostName());
            foreach (var ip in host.AddressList)
            {
                if (ip.AddressFamily == AddressFamily.InterNetwork)
                {
                    Console.WriteLine(ip.ToString());
                }
            }
            Console.OutputEncoding = System.Text.Encoding.UTF8;
            Console.WriteLine("Witaj w świecie C# \uD83D\uDE00\n");
        }
    }
}
